libc/Makefile

```diff
@@ -3,4 +3,10 @@ NAME := test-libc
 SRCS := $(wildcard $(LOCAL_DIR)*.c)
 DEP_LIBS := unity
 
+# TODO: remove after updating docker build image to Ubuntu 22.04
```

Comment:
IMHO `-lpthread` should work everywhere for portability reasons - so I would enable it on all targets.

As for rationale - in POSIX spec (https://pubs.opengroup.org/onlinepubs/9699919799/utilities/c99.html) it is written that `c99` compiler should support this param (not necessarily adding new library to linking process):
![image](https://user-images.githubusercontent.com/3939983/206244372-271df93c-1533-45a4-8e28-b0b4270f0ae6.png)

Reply:
There is the following error then for `ia32-generic-qemu`

![Screenshot from 2022-12-07 18-33-25](https://user-images.githubusercontent.com/77304431/206251007-eac04df3-7487-43b7-8fe6-eb6f1ae5b402.png)

Reply:
so we need to fix it in our buildsystem

Reply:
Thanks, it's now fixed.